### DISCUSSION TITLE
Voice chat will become intermittent when you return to the server lobby

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15023] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Return to the server lobby makes voice chat intermittent and text chat unuseable

### Reproduction steps

1. Start a multiplayer campaign and let other players join
2. Save and return to the server lobby
3. let other players say something

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted using a dedicated server

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I unfortunately could not find a way to get this to happen. Are there any more details you could give on the issue - what do you mean by the voice chat becoming intermittent, how exactly does the text chat become unusable (do the messages just not go through at all)? What kind of mods are you using, are there any that could be modifying the voice or text chat in some way? In any case, I'm sorry for the inconvenience!

**Reply by A2ur3** [reply]
I doubt the issue is related to #14995 where game keeps downloading campaign save
voice chat becoming intermittent means voice transmission continuously got cut off for a split second (the same situation when you start a round when someone is talking meanwhile)
text chat becoming unusable means I can type and send but it wont go through (appear in message area)
we do have mod Soundproof Walls that modifies voice chat but it should not affect voice chat in server lobby
